cmake/apple/install_apple.sh.in

```diff
@@ -21,20 +21,16 @@ MSG_COLOR='\x1B[1;36m'
 COLOR_RESET='\x1B[0m'
 echo -e "$MSG_COLOR\n\nCreating App Bundle \"$APP\"...$COLOR_RESET"
 
-# Locate macdeployqt, assume homebrew & Qt5
-which macdeployqt > /dev/null 2>&1
```

Comment:
So we're making the decision to pull MacPorts support? (that is what `which macdeployqt` did).